### PR TITLE
DAT-2812 add custom theming

### PR DESCRIPTION
**Pull request by idradm**

### CODE REVIEW

File: extensions/wikia/PortableInfobox/templates/PortableInfoboxWrapper.mustache [hunk 1]
```diff
@@ -1 +1 @@
-<aside class="portable-infobox">{{{content}}}</aside>
\ No newline at end of file
+<aside class="portable-infobox {{{theme}}}">{{{content}}}</aside>
```
Comment (SebastianMarzjan) [inline extensions/wikia/PortableInfobox/templates/PortableInfoboxWrapper.mustache]: I wonder if passing {{{classes}}} would be clearer and help avoid the whitespace in markup below

File: extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php [hunk 2]
```diff
@@ -3,6 +3,8 @@
 class PortableInfoboxParserTagController extends WikiaController {
 	const PARSER_TAG_NAME = 'infobox';
 	const INFOBOXES_PROPERTY_NAME = 'infoboxes';
+	const DEFAULT_THEME_NAME = 'wikia';
+	const INFOBOX_THEME_SUFFIX = "-infobox-theme";
```
Comment (SebastianMarzjan) [inline extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php]: Let's make it into a suffix.

**Comment by SebastianMarzjan** [comment]
Test failure unrelated, lgtm

**Comment by RafLeszczynski** [comment]
LGTM

**Pull request merged**